### DISCUSSION TITLE
Bad Request (400) when adding a domain name ?

### DISCUSSION THREAD

**Discussion by Jason-MK** [netbox-community/netbox#7312] [General]
Hello, 

I get this error Bad Request (400) 

When I tried to add a domain name to my IP address ? 

is there any help please ? 

Best Regards

**Comment by candlerb** [comment]
You need to provide more details.  Is this a web request, or an API request?  If it's an API request, show the complete request and response (the response body will include details of the problem).  If it's a web request, show what you see on screen.  If you're generating the request using code, e.g. with pynetbox or curl, show your code.  etc. etc.

**Reply by Jason-MK** [reply]
All what I did is I did add a domain name to the IP of the Netbox ?? I mean let's say 

192.168.100.100 to netbox.github.com 

then I see this error

**Reply by candlerb** [reply]
If I understand this correctly,  you find you can't access the Netbox web interface using http://netbox.github.com even though netbox.github.com maps to the correct IP address.

If that's the case, then you need to edit `configuration.py`, add `netbox.github.com` to the `ALLOWED_HOSTS` setting, and restart netbox.

e.g.

```
ALLOWED_HOSTS = ['192.168.100.100', 'netbox.github.com']
```

**Reply by Jason-MK** [reply]
Thanks a lot man  you are the best
